### PR TITLE
[DJFS-37]Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Database Design For A Transportation Company
-I have created three(3) tables and linked them,the tables are:
**Passengers**
**Trip-details**
**Accidents**

### CODE REVIEW

File: submissions/chabuChabu/database/trips.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*The following query is used to create a table for storing passanger details*/
+use trips;
+create table passangers(
+id int not null auto_increment,
+fullname varchar(100) not null,
+sex varchar(10) not null,
+age int(11),
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:7]: use `varchar` instead

File: submissions/chabuChabu/database/trips.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+passanger_id int not null,
+primary key(id),
+foreign key(passanger_id) references passangers(id)
+);
+
+/*The following query is used to create a table for storing information about accidents*/
+/*It references the passanger table via passanger ID*/
+create table accidents(
+id int not null auto_increment,
+status enum ('survived', 'died') not null,
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:33]: use `tinyint` instead

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/database/trips.sql (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/database/trips.sql:7]: @adaugochi which line are you referring too?

File: submissions/chabuChabu/database/trips.sql (hunk 2 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/database/trips.sql:33]: @adaugochi which line are you referring too??

File: submissions/chabuChabu/database/trips.sql (hunk 1 above)
Comment (adaugochi) [reply submissions/chabuChabu/database/trips.sql:7]: Line 7 @ChabuChabu

File: submissions/chabuChabu/database/trips.sql [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/*The following query is used to create a table for storing passanger details*/
+use trips;
+create table passangers(
+id int not null auto_increment,
+fullname varchar(100) not null,
+sex varchar(10) not null,
+age tinyint(11),
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:7]: I said you should make use of varchar not ringing

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/database/trips.sql (hunk 2 above)
Comment (adaugochi) [reply submissions/chabuChabu/database/trips.sql:33]: line 33

Comment (adaugochi) [reply submissions/chabuChabu/database/trips.sql:33]: Already know that it is the line that I comment under that I am referring to. That's how comments are dropped on PR @ChabuChabu

File: submissions/chabuChabu/database/trips.sql (hunk 1 above)
Comment (adaugochi) [reply submissions/chabuChabu/database/trips.sql]: Line 7

File: submissions/chabuChabu/database/trips.sql (hunk 2 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/database/trips.sql:33]: @adaugochi All right that's noted

File: submissions/chabuChabu/database/trips.sql [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+/*The following query is used to create a table for storing trip details*/
+/*It references the passanger table  via passanger ID*/
+create table trip_details(
+id int not null auto_increment,
+passanger_class int(11) not null,
+ticket_number varchar (50) not null,
+trip_fare decimal(10,2)not null,
+cabin varchar (50),
+parents int (11),
+children int (11),
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:20]: This supposes to be `no_of_parent_or_children` as a single column. Not having `parents` and `children` as a separate column

File: submissions/chabuChabu/database/trips.sql [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+spouses int (11),
+siblings int (11),
```
Comment (adaugochi) [inline submissions/chabuChabu/database/trips.sql:22]: Same goes for `spouses` and `siblings`. This should not be two separate columns but one

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**